nano/node/bootstrap.cpp

```diff
@@ -743,6 +743,129 @@ void nano::bulk_push_client::push_block (nano::block const & block_a)
 	});
 }
 
+nano::bulk_pull_account_client::bulk_pull_account_client (std::shared_ptr<nano::bootstrap_client> connection_a, nano::account const & account_a) :
+connection (connection_a),
+account (account_a),
+total_blocks (0)
+{
+	std::lock_guard<std::mutex> mutex (connection->attempt->mutex);
+	connection->attempt->condition.notify_all ();
+}
```

Comment:
No need to lock over the notify here right? (pessimization)

Reply:
Seems correct